packages/cli/lib/errorHandlers/apiErrors.js

```diff
@@ -3,7 +3,7 @@ const { getAccountConfig } = require('@hubspot/local-dev-lib/config');
 const {
   isMissingScopeError,
   isApiUploadValidationError,
-  getAxiosErrorWithContext,
+  getHubSpotHttpErrorWithContext,
```

Comment:
I know this is constantly in flux, but the current version of this in local-dev-lib is `getUserFriendlyHttpErrorMessage`